### ISSUE TITLE
on break event problemly

### ISSUE THREAD

**Issue by OverBrave** [SkriptLang/Skript#2755]
### Errors / Screenshots

```
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[14:40:02] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[14:40:02] [Server thread/ERROR]: #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
[14:40:02] [Server thread/ERROR]: #!#! Here is full list of them:
[14:40:02] [Server thread/ERROR]: #!#! skript-mirror v2.0.0-SNAPSHOT (https://github.com/btk5h/skript-mirror) Vixio v2.0.4 (http://vixio.space) TuSKe v1.8.2-Pikachu-Patch-3 (github.com/Tuke-Nuke/TuSKe) skript-yaml v1.3.2 SkQuery v4.1.1 
[14:40:02] [Server thread/ERROR]: #!#! We could not identify which of those are specially related, so this might also be Skript issue.
[14:40:02] [Server thread/ERROR]: #!#! You should try disabling those plugins one by one, trying to find which one causes it.
[14:40:02] [Server thread/ERROR]: #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
[14:40:02] [Server thread/ERROR]: #!#! In that case, you will be given instruction on how should you report it.
[14:40:02] [Server thread/ERROR]: #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
[14:40:02] [Server thread/ERROR]: #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! Stack trace:
[14:40:02] [Server thread/ERROR]: #!#! java.lang.UnsupportedOperationException
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.util.SimpleExpression.change(SimpleExpression.java:239)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.util.ConvertedExpression.change(ConvertedExpression.java:153)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.effects.EffHealth.execute(EffHealth.java:110)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:52)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:57)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:156)
[14:40:02] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler$1.execute(SkriptEventHandler.java:114)
[14:40:02] [Server thread/ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[14:40:02] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[14:40:02] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.PlayerInteractManager.breakBlock(PlayerInteractManager.java:313)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:272)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:240)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1319)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.PacketPlayInBlockDig.a(SourceFile:40)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.PacketPlayInBlockDig.a(SourceFile:10)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.MinecraftServer.aZ(MinecraftServer.java:1037)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1030)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1014)
[14:40:02] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:937)
[14:40:02] [Server thread/ERROR]: #!#!     at java.lang.Thread.run(Thread.java:748)
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! Version Information:
[14:40:02] [Server thread/ERROR]: #!#!   Skript: 2.5-alpha1 (latest)
[14:40:02] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
[14:40:02] [Server thread/ERROR]: #!#!     Date: 2020-01-11
[14:40:02] [Server thread/ERROR]: #!#!   Bukkit: 1.15.1-R0.1-SNAPSHOT
[14:40:02] [Server thread/ERROR]: #!#!   Minecraft: 1.15.1
[14:40:02] [Server thread/ERROR]: #!#!   Java: 1.8.0_222 (OpenJDK 64-Bit Server VM 25.222-b10)
[14:40:02] [Server thread/ERROR]: #!#!   OS: Linux amd64 3.10.0-1062.4.1.el7.x86_64
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! Server platform: Paper
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! Current node: null
[14:40:02] [Server thread/ERROR]: #!#! Current item: damage (the tool of the player >> ChainedConverter{first=ch.njol.skript.classes.data.DefaultConverters$18@3e0973ee,second=ch.njol.skript.classes.data.DefaultConverters$15@3183d884}: slot -> itemstack -> itemstack -> itemtype) by [[long:3]]
[14:40:02] [Server thread/ERROR]: #!#! Current trigger: break of diamond ore (break/place/burn/fade/form of diamond ore) (sctas.sk, line 25)
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! Thread: Server thread
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! Language: english
[14:40:02] [Server thread/ERROR]: #!#! Link parse mode: DISABLED
[14:40:02] [Server thread/ERROR]: #!#! 
[14:40:02] [Server thread/ERROR]: #!#! End of Error.
[14:40:02] [Server thread/ERROR]: #!#! 
```

### Server Information
* **Server version/platform:**       1.15.1
* **Skript version:**                2.5-alpha1

**Comment by ShaneBeee** [comment]
what is the code you are using that throws this error?

**Comment by OverBrave** [comment]
> on break of diamond ore:
> 	if player's tool is not pickaxe:
> 		cancel event
> 	else:
> 		cancel event
> 		if {sctaskaz::%player%} is set:
> 			damage player's tool by 0

**Comment by ShaneBeee** [comment]
I can replicate this issue.

**Comment by ShaneBeee** [comment]
Fixed in 2.5-alpha2

**Comment by ShaneBeee** [comment]
Re-opening this. Closed it by accident.
This issue has been fixed in bensku's PR, but it hasn't been merged yet.
